task/sql-tasks.js

```diff
@@ -178,7 +287,16 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT

```

Comment:
it's better to use 2 select statements and one select without from clause. From - clause is optional
eg
```
select
 (select count(*) from ....) as 'Total'
 (select count(*) from ....) as 'TotalDisc'
```